.github/workflows/test.yml

```
@@ -1,9 +1,16 @@
 name: Test
 
-on: [push, pull_request]
+on:
+  push:
+  # Triggers the workflow on labeled PRs only.
+  pull_request_target:
+    types: [ labeled ]
+  schedule:
+    - cron: '0 6 * * *' # Runs every day at 6:00 AM GMT
 
 jobs:
   test:
+    if: github.event_name == 'schedule' || github.event_name == 'push' || (github.event_name == 'pull_request' && contains(github.event.pull_request.labels.*.name, 'safe to test'))
```

Prevent running schedule on forks
```suggestion
    if: (github.event_name == 'schedule' && github.repository_owner == 'jfrog') || github.event_name == 'push' || (github.event_name == 'pull_request' && contains(github.event.pull_request.labels.*.name, 'safe to test'))
```